src/share.ts

```diff
@@ -178,7 +178,7 @@ async function sendChunkedResults(evalRecord: Eval, url: string): Promise<string
   logger.debug(`Largest result size from sample: ${largestSize} bytes`);
 
   // Determine how many results per chunk
-  const TARGET_CHUNK_SIZE = 10 * 1024 * 1024; // 10MB in bytes
+  const TARGET_CHUNK_SIZE = 0.9 * 1024 * 1024; // 900KB in bytes
```

Comment:
Do we already warn the users to set the chunk size var if it fails? I feel like I prefer to keep the default high since most users presumably don’t run into this issue and sharing is faster?

Reply:
In my recent anecdotal experience sharing frequently fails, so I'd much rather wait longer

Reply:
Fair enough, just find it odd that this seems to be a recent thing